diagnostic.md

```diff
@@ -19,7 +23,8 @@ c = c + d
 ```
 
 After this code executes, what is the value of c?  Please identify the operators in the last line of this program `c = c + d` and explain what they do.
-
+AFTER the code executes, c = 7. It executes from right to left, so the
+expression would be c = 5 + 2, which equates to 7.
```

Comment:
Close enough here. Specifically we wanted you to say that the `+` is adding together 5 and 2, and the `=` is assigning the result to `c`. Sounds like you know that though.